src/main/java/org/jabref/logic/crawler/LibraryEntryToFetcherConverter.java

```diff
@@ -8,19 +8,16 @@
 import org.jabref.logic.importer.ImportFormatPreferences;
 import org.jabref.logic.importer.SearchBasedFetcher;
 import org.jabref.logic.importer.WebFetchers;
-import org.jabref.model.entry.BibEntry;
-import org.jabref.model.entry.field.UnknownField;
-
-import static org.jabref.model.entry.types.SystematicLiteratureReviewStudyEntryType.LIBRARY_ENTRY;
+import org.jabref.model.study.LibraryEntry;
 
 /**
  * Converts library entries from the given study into their corresponding fetchers.
  */
 class LibraryEntryToFetcherConverter {
-    private final List<BibEntry> libraryEntries;
+    private final List<LibraryEntry> libraryEntries;
```

Comment:
Can you use this converter already during parsing with jackson, so that a study already contains a list of fetcher, making the class `LibraryEntry` obsolete?